documentation/docs/mandates.mdx

```diff
@@ -27,9 +27,8 @@ Mandates represents delegated access resource to a users account.
 ## APIs
 
 ### Create
-
-A Client creates a Mandate by doing an HTTP POST of JSON document to the Mandates endpoint as defined in the Open
-Payments Metadata.
+Given a sender's payment identifier (e.g., a Payment Pointer), a Client creates a Mandate by doing an HTTP POST of JSON document to the 
```

Comment:
Whose asset are mandates be denominated in? Are how do exchange rate fluctuations affect them?

If I as a sender am authorizing a recurring mandate, my local currency is USD, but receiver wants to pull in BTC, I probably don't want price exposure to BTC? How should senders cap the amount that can be pulled?